### PR TITLE
Add static asset service user docs

### PR DESCRIPTION
**Pull request by cstns**
## Description

Add static asset service user docs

## Related Issue(s)

none

## Checklist

 - [x] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [ ] Suitable unit/system level tests have been added and they pass 
 - [ ] Documentation has been updated
    - [ ] Upgrade instructions
    - [ ] Configuration details
    - [ ] Concepts
 - [ ] Changes `flowforge.yml`?
    - [ ] Issue/PR raised on `FlowFuse/helm` to update ConfigMap Template
    - [ ] Issue/PR raised on `FlowFuse/CloudProject` to update values for Staging/Production

## Labels

 - [ ] Includes a DB migration? -> add the `area:migration` label

### CODE REVIEW

**Comment by joepavitt** [comment]
Any reason for the draft? Or can I review?

**Comment by cstns** [comment]
There's an open question on dpt eng about some points in the docs that would need clarifying

**Comment by cstns** [comment]
about the limitations section*

**Comment by hardillb** [comment]
@cstns how are we doing with this?

**Comment by cstns** [comment]
I need to update the limitations section. 
I have an answer for Unsupported characters, which ill make a vague remark about depending on file system, but the other two remain unanswered:
	- Allowed MIME types: // todo couldn't find any on the BE or localfs driver
	- Team permissions required: owner // to be confirmed or rephrased

**Comment by hardillb** [comment]
We do not care about mime types for the currently deployed code, everything is just uploaded as binary blob.

We may want to think about mime types for when sharing is enabled, but that is part of the next release (and we are going to default to what Node-RED already does).

**Comment by hardillb** [comment]
Only Owner and Members should be able to access the Assests tab.

**Comment by cstns** [comment]
As we stand, viewer type roles have access to the assets tab, but cannot retrieve assets so the list is always empty. The asset action buttons are still enabled (refresh, new folder and upload).

Should I create another issue or do a quick fix in this one?

I can also remove the mime type restriction from the docs until we implement it.

**Comment by hardillb** [comment]
Raise an issue for the Viewer stuff.

Remove the mime stuff

**Comment by cstns** [comment]
linking https://github.com/FlowFuse/flowfuse/issues/4439

File: docs/user/static-asset-service.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
+---
+navTitle: Static asset service
+navOrder: 1
+---
+
+# Static asset service
+
+Our platform now includes a Static Asset Service, enabling you to save, access, and manage files seamlessly within your hosted Node-RED instances.
```
Comment (hardillb) [inline docs/user/static-asset-service.md:8]:
```suggestion
Our platform now includes a Static Asset Service, enabling you to manage files seamlessly within your hosted Node-RED instances.
```

Removing access and leaving it as just manage for now.

File: docs/user/static-asset-service.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+## What is Static asset service?
+
+The Static Asset Service allows you to store files permanently within your FlowFuse Instances. Files that you upload, generate, or modify will remain accessible even after your session ends or the application restarts.
+
+## Prerequisites
+
+### FlowFuse Cloud
+
+- A launcher version of 2.8.0 or greater.
```
Comment (hardillb) [inline docs/user/static-asset-service.md:18]:
```suggestion
- A Instance Stack with a launcher version of 2.8.0 or greater.
```

**Review by hardillb — CHANGES_REQUESTED** [review]
Also probably need a link from docs/users/index.md in the "Working with Files and Context" section

**Comment by hardillb** [comment]
Raised follow up https://github.com/FlowFuse/flowfuse/issues/4440 to clean up file node docs.

**Review by hardillb — APPROVED** [review]

**Pull request merged**